File: src/main/java/top/zbeboy/isy/web/bean/graduate/design/pharmtech/GraduationDesignTutorBean.java
package top.zbeboy.isy.web.bean.graduate.design.pharmtech;

import lombok.Data;
import lombok.EqualsAndHashCode;
import top.zbeboy.isy.domain.tables.pojos.GraduationDesignTutor;

/**
 * Created by lenovo on 2017-05-20.
 */
@EqualsAndHashCode(callSuper = false)
@Data
public class GraduationDesignTutorBean extends GraduationDesignTutor {
    private String realName;
    private String mobile;
}

File: src/main/java/top/zbeboy/isy/web/bean/graduate/design/pharmtech/GraduationDesignTutorBean.kt
package top.zbeboy.isy.web.bean.graduate.design.pharmtech

import top.zbeboy.isy.domain.tables.pojos.GraduationDesignTutor

/**
 * Created by zbeboy 2017-12-20 .
 **/
class GraduationDesignTutorBean : GraduationDesignTutor() {
    var realName: String? = null
    var mobile: String? = null
    var username: String? = null
    var organizeName: String? = null
    var studentNumber: String? = null
    var studentName: String? = null
    var staffName: String? = null
    var graduationDesignReleaseId: String? = null
    var staffId: Int = 0
}